tests/codegen/paginator-tests/build.gradle.kts

```diff
@@ -65,6 +65,7 @@ dependencies {
 
     compileOnly(project(":smithy-kotlin-codegen"))
     implementation(project(":runtime:runtime-core"))
+    implementation(project(":runtime:smithy-client"))
```

Comment:
**Question**: Do we need to explicitly depend on **runtime-core** if **smithy-client** pulls it in as an API dependency?

Reply:
Technically no but it harms nothing to have it this way.